images/metacontroller/tests/02-helm.sh

```diff
@@ -198,7 +196,7 @@ EOF
 
 kubectl -n hello apply -f hello.yaml
 
-# Sleep  theh ello controller kicks in 
+# Sleep  theh ello controller kicks in
```

Comment:
```suggestion
# Sleep the hello controller kicks in
```

Reply:
I think merging this will clear my LGTM, and this PR is merge conflict catnip, so let's merge this as-is and follow up.

Reply:
Bah I committed before reading your comment. If you're around to approve again cool, otherwise I'll deal with the consequences tomorrow.